src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let result = 0;
+  const oneRentalDay = 40;
+  const total = days * oneRentalDay;
+
+  if (days > 0 && days < 3) {
+    result = total;
+  } else if (days >= 3 && days < 7) {
+    result = total - 20;
+  } else if (days >= 7) {
+    result = total - 50;
```

Comment:
move discount to variables. Dontuse else 
![2023-01-18_15h13_34](https://user-images.githubusercontent.com/61904995/213180849-5f30e7c1-fa6c-45b1-a92b-655b610acb93.png)